extensions/r/package.json

```diff
@@ -3,9 +3,6 @@
 	"version": "0.1.0",
 	"publisher": "vscode",
 	"engines": { "vscode": "*" },
-	"scripts": {
```

Comment:
This syntax based on https://github.com/randy3k/R-Box/blob/master/syntax/R%20Extended.sublime-syntax and that is using `sublime-syntax` instead of `tmbundle`. I think "update-grammar" not support this.